net-analyzer/gvmd/gvmd-20.8.1.ebuild

```diff
@@ -0,0 +1,126 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake flag-o-matic systemd toolchain-funcs
+
+DESCRIPTION="Greenbone vulnerability manager, previously named openvas-manager"
+HOMEPAGE="https://www.greenbone.net/en/"
+SRC_URI="https://github.com/greenbone/gvmd/archive/v${PV}.tar.gz -> ${P}.tar.gz
+		https://github.com/j-licht/gvmd_report_formats/archive/v0.1.tar.gz -> gvm-report-formats-0.1.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2+"
+KEYWORDS="~amd64 ~x86"
+IUSE="extras +postgres sqlite test"
+# keep deprecated USE Flags for revdep of gvm
+REQUIRED_USE="postgres !sqlite"
```

Comment:
Really these `postgres` and `sqlite` USE flags make no sense to me, especially since you depend on *dev-db/postgresql* unconditionally.

gvm is a metapackage and even the latest version has dropped these deps. Could you get rid of them?